addons/mail/models/res_company.py

```diff
@@ -23,18 +30,31 @@ class Company(models.Model):
         readonly=False, store=True)
 
     @api.depends('name')
+    def _compute_alias_domain_id(self):
+        toupdate = self.filtered(lambda company: not company.alias_domain_id)
+        if toupdate:
+            alias_domain = self.env['mail.alias.domain'].search([], limit=1)
+            toupdate.alias_domain_id = alias_domain.id
+
+    @api.depends('alias_domain_id')
+    def _compute_bounce(self):
+        self.bounce_email = ''
+        self.bounce_formatted = ''
+
+        for company in self.filtered('alias_domain_id'):
```

Comment:
In previous method those are called "toupdate", we could align everything to "company" 👀